README.md

```diff
@@ -20,9 +20,9 @@ Cuốn sách này được dịch và đăng tại https://d2l.aivivn.com/.
 * [x] Giới thiệu
 * [ ] 2. Preliminaries
     * [-] 2.1. Data Manipulation
-    * [ ] 2.2. Data Preprocessing
+    * [-] 2.2. Data Preprocessing
     * [-] 2.3. Đại số Tuyến tính
-    * [ ] 2.4. Calculus
+    * [-] 2.4. Calculus
```

Comment:
Em cố gắng điều phối sao cho càng ít phần có `[-]` càng tốt. Tập trung đến đâu xong đến đấy nhé.

@duythanhvn

Reply:
Vâng anh ạ, từ bây giờ mình đi theo thứ tự để ở README. Xong từng phần mình sẽ triển khai revise toàn chương ạ.